float32 rover_x
float32 rover_y
float32 rover_theta
float32 hit_point_x
float32 hit_point_y
bool hit
